.devops/package.nix

```diff
@@ -0,0 +1,119 @@
+{ lib
+, config
+, stdenv
+, cmake
+, ninja
+, pkg-config
+, git
+, python3
+, openmpi
+, openblas
```

Comment:
Nixpkgs also provides the "shim" attributes `blas` and `lapack` which generate implementation agnostic pkg-config files, but let the user switch between openblas, mkl, etc.

This one I'm not sure will work out of the box, but you could leave a comment about trying the generic `blas` instead of `openblas` in future

Reply:
You were right. I added a comment.